diff --git a/src/track/keyutils.cpp b/src/track/keyutils.cpp
--- a/src/track/keyutils.cpp
+++ b/src/track/keyutils.cpp
@@ -46,7 +46,7 @@
         QStringLiteral(u"E♭"),
         QStringLiteral(u"E"),
         QStringLiteral(u"F"),
-        QStringLiteral(u"F♯/G♭"),
+        QStringLiteral(u"F♯"),
         QStringLiteral(u"G"),
         QStringLiteral(u"A♭"),
         QStringLiteral(u"A"),
@@ -55,7 +55,7 @@
         QStringLiteral(u"Cm"),
         QStringLiteral(u"C♯m"),
         QStringLiteral(u"Dm"),
-        QStringLiteral(u"D♯m/E♭m"),
+        QStringLiteral(u"D♯m"),
         QStringLiteral(u"Em"),
         QStringLiteral(u"Fm"),
         QStringLiteral(u"F♯m"),
